code/src/java/plugin/overland/model/TravelMethodFactory.java

```diff
@@ -89,7 +95,7 @@
 		if (path.isDirectory())
 		{
 			File[] dataFiles = path.listFiles(new XMLFilter());
-			SAXBuilder builder = new SAXBuilder();
+			SAXEngine builder = new SAXBuilder();
```

Comment:
Don't understand the semantic value to a reader here, all I see is a tool saying "do this" without applying thought to the process.